packages/design-system-react/src/components/button-primary/ButtonPrimary.stories.tsx

```diff
@@ -0,0 +1,180 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+import { ButtonPrimarySize } from '.';
+import { IconName } from '..';
+import { ButtonPrimary } from './ButtonPrimary';
+import README from './README.mdx';
+
+const meta: Meta<typeof ButtonPrimary> = {
+  title: 'React Components/ButtonPrimary',
+  component: ButtonPrimary,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    children: {
+      control: 'text',
+      description:
+        'Required prop for the content to be rendered within the ButtonPrimary',
+    },
+    className: {
+      control: 'text',
+      description:
+        'Optional prop for additional CSS classes to be applied to the ButtonPrimary component',
+    },
+    isDanger: {
+      control: 'boolean',
+      description:
+        'Optional prop that when true, applies error/danger styling to the button',
+    },
+    isInverse: {
+      control: 'boolean',
+      description:
+        'Optional prop that when true, applies inverse styling to the button',
+    },
+    size: {
+      control: 'select',
+      options: Object.values(ButtonPrimarySize),
```

Comment:
last we talked, I remember you not liking using Object.values due to the values in some cases being tailwind classnames. I remember that I tried using just ButtonPrimarySize and it still works. wdyt?

Reply:
Great catch! I'll update in a subsequent clean up PR that tackles this issue in all of the button stories! Issue here https://github.com/MetaMask/metamask-design-system/issues/306